modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -145,22 +145,22 @@ RETURN n.name
 [[select-default-cypher-version]]
 == Select the default Cypher version when creating a database
 
-Databases created on Neo4j 2025.05 or later will have Cypher 25 as their default language (if {neo4j-docs-base-uri}/operations-manual/current/configuration/configuration-settings/#config_db.query.default_language[`config_db.query.default_language`] is set to `CYPHER_25` -- see  xref:queries/select-version.adoc#config-database-upgrades[Cypher versions, configuration settings, and DBMS upgrades] below).
-This is true for the link:{neo4j-docs-base-uri}/operations-manual/current/database-administration/#manage-database-systems[system, standard, and composite] Neo4j databases.
+Databases created on Neo4j 2025.06 or later will continue to have Cypher 5 as their default language (if {neo4j-docs-base-uri}/operations-manual/current/configuration/configuration-settings/#config_db.query.default_language[`config_db.query.default_language`] is set to `CYPHER 5` -- see  xref:queries/select-version.adoc#config-database-upgrades[Cypher versions, configuration settings, and DBMS upgrades] below).
```

Comment:
```suggestion
Databases created on Neo4j 2025.06 or later will continue to have Cypher 5 as their default language (if {neo4j-docs-base-uri}/operations-manual/current/configuration/configuration-settings/#config_db.query.default_language[`config_db.query.default_language`] is set to `CYPHER_5` -- see  xref:queries/select-version.adoc#config-database-upgrades[Cypher versions, configuration settings, and DBMS upgrades] below).
```
I think the config setting has `_` so when referring to setting that we might want it but not otherwise?